cmd/data-scanner.go

```diff
@@ -246,6 +246,8 @@ type folderScanner struct {
 	healObjectSelect      uint32 // Do a heal check on an object once every n cycles. Must divide into healFolderInclude
 	scanMode              madmin.HealScanMode
 
+	fullThrottle func() bool
```

Comment:
```suggestion
	weSleep func() bool
```

is better @vadmeste which returns `true` when `cap(tokens) != len(tokens)`

Reply:
PTAL @vadmeste